PluginsAndFeatures/azure-toolkit-for-intellij/rider/ReSharper.Azure/src/Azure.Intellisense/FunctionApp/LiveTemplates/RiderAzureCSharpFileTemplatesOptionPage.cs

```diff
@@ -0,0 +1,47 @@
+﻿// Copyright (c) 2020 JetBrains s.r.o.
+// <p/>
```

Comment:
Do we still use this `</p>` paragraph separator in header?

Reply:
For C# side seems we do. Might make sense to do a global find/replace at some point.